docs/api/Components.md

```diff
@@ -182,31 +157,29 @@ describe('TestComponent', function () {
             // React must be required after window is set
             React = require('react');
```

Comment:
@mridgway the tool i am suggesting is to remove all setup required by jsdom + ReactTestUtils.